generate.js

```diff
@@ -46,6 +48,8 @@ const template = `
   </head>
   <body>
     <h1>GCI Current Leaders</h1>
+    <small id="last-generated"></small>
```

Comment:
@Indmind ^ Please do that.

You can use `data` attribute here i.e. `data-time` and then get that value in js with [`.dataset`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset)

Reply:
where should I put the static folder?

Reply:
Your placement is already correct

Reply:
it doesn't work, if i move static folder to out folder, it works

Reply:
Hmm I see.
https://github.com/coala/gci-leaders/pull/31 introduced that behaviour of copying static to out folder.
It'll be merged soon.

You can do/test the other stuff while waiting.